File: C# Tutorials/Loops/DoWhile.cs
/* Do_While.cs
Add a list of integers. Keep going until End Of File
Written by: Your_Name
Date:
*/

using System;

public class DoWhile
{
        static void Main ()
        {	
	/* Local Declarations */
	int iUser_Input ;
	int iSum = 0 ;

	/* Statements */
        Console.WriteLine ( "Enter some numbers ( 0 to stop ) :\n" ) ; 

	do
		{
		iUser_Input = int.Parse ( Console.ReadLine () ) ;
		iSum += iUser_Input ;
		Console.WriteLine ( "Total: {0}", iSum ) ;
		}
	while ( iUser_Input != 0 ) ;

	Console.WriteLine ( "\nProgram terminated" ) ; 
	Console.WriteLine ( "Have a day. :| \n" ) ; 
	}	// end Main
}	// end class DoWhile

File: Programming4NE1/CPlusPlus/Source/Do_While.cpp
/* Do_While.cpp
Add a list of integers.
Keep going until End Of File
Written by: Your_Name
Date:
*/

#include <iostream>
using namespace std ; 

int main ( void )
{
/* Local Declarations */
	int User_Input ;
	int sum = 0 ;

/* Statements */
	cout << "Enter some numbers ( 0 to stop ) : " << endl ;

	do
{
        cin >> User_Input ; 
	sum += User_Input ;
	cout << "Total: "<< sum  << endl ;
}
	while ( User_Input != 0 ) ;

	cout << endl << "Program terminated " <<endl ;
	cout << "Have a day. :|" << endl ; 
return 0 ;
}	/* main */
